### PR TITLE
Change gamification routes

### PR DESCRIPTION
**Pull request by stanislav-grin**

### CODE REVIEW

File: src/gamification.js [hunk 1]
```diff
@@ -1,28 +1,32 @@
 import urls from './urls'
 
 export default req => ({
-  loadTracks(appId) {
-    return req.get(`${urls.gamification(appId)}/tracks`)
+  loadTracks() {
+    return req.get(`${urls.console()}/gamification/tracks`)
```
Comment (Valodya) [inline src/gamification.js:5]: I the MR I don't see you removed `urls.gamification`, or it's already removed?

Comment (Valodya) [reply src/gamification.js:5]: oh, I see it below, 

what's a difference between the following:
- `${urls.console()}/gamification/....`
- `${urls. gamification(appId)}/....`

Comment (stanislav-grin) [reply src/gamification.js:5]: The difference is that some routes doesn't requiring appId for data to be loaded, while other is requiring appId to be present

Comment (Valodya) [reply src/gamification.js:5]: does make sense  to change

```
export const gamification = appId => `${appId ? appConsole(appId) : console()}/gamification`
```
?

Comment (stanislav-grin) [reply src/gamification.js]: Done

**Pull request merged**